pelican/tools/templates/fabfile.py.in

```diff
@@ -3,7 +3,10 @@ import fabric.contrib.project as project
 import os
 import shutil
 import sys
-import SocketServer
+if sys.version_info[0] >= 3:
```

Comment:
I read it wrong, they do propose the new version under try :) Nevermind about that then, I guess I'll just accept your change and have it be the official way.